### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by tmoor039**
Fully tested.. need to hash passCof for login.. 

Please review.. Creates users .. ready to deploy!

### CODE REVIEW

File: Code/webapp/app.js [hunk 1]
```diff
@@ -4,5 +4,6 @@ angular.module('mainApp', [
     'vip-projects',		//Features Module
 	'ProjectProposalController',
 	'routes',
-	'projectApplicationController'
+	'userRegistrationController',
```
Comment (ghost) [inline Code/webapp/app.js:8]: dont think you merged properly, this 'projectApplicationController' dependency was removed

Comment (tmoor039) [inline Code/webapp/app.js:8]: that dependency its still currently on the master branch @mrowe009

Comment (ghost) [inline Code/webapp/app.js:8]: and itll get removed as soon as this gets merged

Comment (tmoor039) [inline Code/webapp/app.js:8]: fixed

**Comment by ghost** [comment]
I approve, but need someone else to check it.

**Comment by Mcond005** [comment]
i approve

**Pull request merged**